src/main/java/com/google/cloud/genomics/utils/grpc/Example.java

```diff
@@ -35,11 +34,11 @@ public static void main(String[] args) throws Exception {
       return;
     }
     
-    Channel channel = Channels.fromOfflineAuth(auth);
+    GenomicsChannel channel = GenomicsChannel.fromOfflineAuth(auth);
 
     // Regular RPC example: list all reference set assembly ids.
     ReferenceServiceV1BlockingStub refStub =
-        ReferenceServiceV1Grpc.newBlockingStub(channel);
+        ReferenceServiceV1Grpc.newBlockingStub(channel.getChannel());
```

Comment:
channel.getChannel() in particular feels pretty awkward. Maybe it would be better to have GenomicsChannel extend Channel so it could be used directly as a gRPC channel. There is only one abstract method which you can delegate to the gRPC channel you are wrapping.